monetio/models/raqms.py

```diff
@@ -52,10 +74,27 @@ def open_mfdataset(fname, *, surf_only=False):
             "in netCDF format."
             "Do not mix and match file types."
         )
-
-    ds = xr.open_mfdataset(names, concat_dim="time", drop_variables=["theta"], combine="nested")
+    if var_list is not None:
+        var_list.extend(['lat','lon',"IDATE", "Times","psfc","delp",'pdash','ttheta'])
+        ds = xr.open_mfdataset(names, concat_dim="time", drop_variables=["theta"], combine="nested")[var_list]
+    else:
+        ds = xr.open_mfdataset(names, concat_dim="time", drop_variables=["theta"], combine="nested")
     ds = _fix(ds, surf_only=surf_only)
-
+    
+    # convert all gas species to ppbv
+    if convert_to_ppb:
+        for i in ds.variables:
+            if "units" in ds[i].attrs:
+                if "ppv" in ds[i].attrs["units"]:
+                    ds[i] *= 1e9
+                    ds[i].attrs["units"] = "ppbv"
+    
+    if 'ttheta' in ds.keys():
+        # Calculate temperature from potential temperature
+        k = 0.28571428571428564 # metpy.constants kappa
+        ds['temperature_k'] = ds['ttheta']*(ds['pres_pa_mid']/100000)**k
+        ds['temperature_k'].attrs['units'] = 'K'
```

Comment:
Do you want to add `long_name` as well?

Reply:
@mbruckner-work if you don't want this, let me know and/or mark resolved and I'll approve the PR.